<ui version="4.0" >
 <class>QuickSettingsWindow</class>
 <widget class="QDialog" name="QuickSettingsWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Quick Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="videoGroupBox" >
     <property name="title" >
      <string>Video</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="contrastLabel" >
        <property name="text" >
         <string>Contrast:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QSlider" name="contrastSlider" >
        <property name="minimum" >
         <number>-8</number>
        </property>
        <property name="maximum" >
         <number>8</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval" >
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="brightnessLabel" >
        <property name="text" >
         <string>Brightness:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QSlider" name="brightnessSlider" >
        <property name="minimum" >
         <number>-8</number>
        </property>
        <property name="maximum" >
         <number>8</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval" >
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="saturationLabel" >
        <property name="text" >
         <string>Saturation:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QSlider" name="saturationSlider" >
        <property name="minimum" >
         <number>-8</number>
        </property>
        <property name="maximum" >
         <number>8</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval" >
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="hueLabel" >
        <property name="text" >
         <string>Hue:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2" >
       <widget class="QSlider" name="hueSlider" >
        <property name="minimum" >
         <number>-8</number>
        </property>
        <property name="maximum" >
         <number>8</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval" >
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QLabel" name="aspectRatioLabel" >
        <property name="text" >
         <string>Aspect Ratio:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QComboBox" name="aspectRatioCombo" >
        <property name="minimumSize" >
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text" >
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Stretch</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>4/3</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>16/9</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0" colspan="2" >
       <widget class="QLabel" name="scaleModeLabel" >
        <property name="text" >
         <string>Scale Mode:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" >
       <widget class="QComboBox" name="scaleModeCombo" >
        <property name="minimumSize" >
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text" >
          <string>Fit in view</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Scale and crop</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="audioGroupBox" >
     <property name="title" >
      <string>Audio</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Audio Device:</string>
          </property>
          <property name="pixmap" >
           <pixmap resource="quarkplayer.qrc" >:/icons/oxygen/16x16/devices/audio-card.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="deviceCombo" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="audioEffectLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Audio Effect:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="audioEffectsCombo" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Maximum" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="effectButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Setup</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="crossFadeLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Cross Fade:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QSlider" name="crossFadeSlider" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum" >
             <number>-20</number>
            </property>
            <property name="maximum" >
             <number>20</number>
            </property>
            <property name="singleStep" >
             <number>1</number>
            </property>
            <property name="pageStep" >
             <number>2</number>
            </property>
            <property name="value" >
             <number>0</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition" >
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="crossFadeLabel1" >
              <property name="font" >
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text" >
               <string>-10 Sec</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="crossFadeLabel2" >
              <property name="font" >
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text" >
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="crossFadeLabel3" >
              <property name="font" >
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text" >
               <string>10 Sec</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="quarkplayer.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QuickSettingsWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QuickSettingsWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
